resources/test_data/tbl/meta_tables/meta_columns.tbl

```diff
@@ -0,0 +1,7 @@
+table|name|type|nullable
```

Comment:
I am wondering whether we should have column_id and data_type in meta_columns as well.

Reply:
Yes to column_id, but data_type is already there?!

Reply:
Right. Should we call it `data_type` instead of `type`?

Reply:
I don't like it, but I don't care enough to discuss it.